code/modules/mob/living/simple_animal/hostile/necro.dm

```diff
@@ -576,3 +576,54 @@
 				set_light(1, 2, "#5dca31")
 
 #undef RAD_COST
+
+
+/////////////////////////////////////////////////////// 
+///////////////// HEADCRAB ZOMBIES //////////////////// 
+///////////////////////////////////////////////////////
+
+/mob/living/simple_animal/hostile/necro/zombie/headcrab //Not very useful
+	icon_state = "zombie_turned" //Looks almost not unlike just a naked guy to potentially catch others off guard
+	icon_living = "zombie_turned"
+	icon_dead = "zombie_turned"
+	desc = "A reanimated corpse that looks like it has seen better days. This one still appears quite human."
+	maxHealth = 75
+	health = 75
+	can_evolve = FALSE
+	canRegenerate = 1
```

Comment:
canRegenerate though. If my guess is right that'd mean they rise up after a while after death

Reply:
Oops, meant to set that to zero